llama_stack/providers/registry/datasetio.py

```diff
@@ -13,10 +13,10 @@ def available_providers() -> List[ProviderSpec]:
     return [
         InlineProviderSpec(
             api=Api.datasetio,
-            provider_type="meta-reference",
+            provider_type="localfs",
             pip_packages=["pandas"],
-            module="llama_stack.providers.inline.meta_reference.datasetio",
-            config_class="llama_stack.providers.inline.meta_reference.datasetio.MetaReferenceDatasetIOConfig",
+            module="llama_stack.providers.inline.datasetio.localfs",
```

Comment:
do we want to have an inline/datasetio/pandas as a provider as well? or are we saying that pandas is the interface between datasetio and rest of the stack?

Reply:
pandas is a library that is used for both huggingface provider and localfs provider.